src/sidebars/sidebar.less

```diff
@@ -187,4 +192,7 @@
     margin-top: 7px;
     margin-right: 0px;
   }
+  .closed-window-selection {
+    display: none;
+  }
```

Comment:
In the original design spec this icon should be moved to the toolbar and resized (#405), but I see that's assigned to @akhmorgan. If he hasn't done that already it looks like something orthogonal to other compact view changes -- maybe you can take it?